.github/workflows/build-all-on-tagging.yml

```diff
@@ -36,7 +36,7 @@ jobs:
   # This workflow contains a single job called "build"
   build-push-image:
     # The type of runner that the job will run on
-    runs-on: ubuntu-21.10
+    runs-on: ubuntu-latest
```

Comment:
Using latest is not a good idea. Having said that it is version used to spin up container for building. Our actual application is not hosted on it.
Also all current versions have vulnerability. 
When  version upgrade for Ubuntu happens we are hoping to get that one.